### ISSUE TITLE
React Native - Warning eslint-import-resolver-typescript has been ignored because it contains invalid configuration

### ISSUE THREAD

**Issue by beetlebum** [alexgorbatchev/eslint-import-resolver-typescript#100]
When using the project in React Native 0.67 and resolver version 2.7, the following error can be observed:
```warn Package eslint-import-resolver-typescript has been ignored because it contains invalid configuration. Reason: Package subpath './package.json' is not defined by "exports" in /Code/GitHub/app/node_modules/eslint-import-resolver-typescript/package.json```

**Comment by JounQin** [comment]
So why `package.json` should be exported for React Native?

**Comment by billnbell** [comment]
this is normal when using exports. just add it in node_modules/eslint-import-resolver-typescript/package.json

  "exports": {
    "./package.json": "./package.json",

**Comment by billnbell** [comment]
```
  "exports": {
    ".": {
      "require": "./lib/index.cjs",
      "import": "./lib/index.es2015.mjs",
      "types": "./lib/index.d.ts"
    },
    "./package.json": "./package.json"
  },
```

**Comment by JounQin** [comment]
I know how to do this, I'm just wandering why `package.json` is required for React Native.

And PR welcome.

**Comment by billnbell** [comment]
https://github.com/alexgorbatchev/eslint-import-resolver-typescript/pull/101

**Comment by JounQin** [comment]
released as v2.7.1
